README.md

```diff
@@ -1,6 +1,55 @@
-# curriculum-cr-training-mobile-menu
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# CR TRAINING: MOBILE MENU
 
 
 This repo host an initial code for the tasks performed during the Code Reviewers training.
 
 Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+
+
```

Comment:
- [ ] **OPTIONAL** It's way too professional if you included a screenshot of your website in the README file.